dojo/tools/awssecurityhub/guardduty.py

```diff
@@ -30,6 +30,7 @@ def get_item(self, finding: dict, test):
             else:
                 mitigated = datetime.utcnow()
         description = f"This is a GuardDuty Finding\n{finding.get('Description', '')}"
+        description += f"\n**AWS Finding ARN: ** {finding_id}\n"
```

Comment:
Could you please also add:
    - GeneratorId --> Into Description

Go through all values please and evaluate which to add.

Reply:
@manuel-sommer done - added GeneratorId into Description.  Did not add for Inspector since I do not have a sample Finding at the moment from that specifically.  If I find one in the future I can add it.